packages/driver/cypress/e2e/commands/net_stubbing.cy.ts

```diff
@@ -2496,6 +2496,30 @@ describe('network stubbing', { retries: 15 }, function () {
         }).visit('/dump-method')
       })
 
+      it('fails test if both req.reply and req.continue are called in req handler', function (done) {
```

Comment:
Just adding an extra test I noticed was missing- there was existing coverage for using `req.reply` multiple times but not for using both `reply` and `continue` in a single handler